spec/fare_transactions.schema.json

```diff
@@ -32,6 +32,12 @@
       }
     },
     {
+       "name": "location_ping_id",
+       "type": "string",
+       "title": "ID referencing vehicle_locations.location_ping_id",
+       "description": "Identifies a vehicle_location that is the location where the fare_transaction occurred"
```

Comment:
I don't think you want to use "vehicle_location" when you mean a generic "vehicle location". Maybe `Identifies a vehicle location associated with this fare transaction`?

Reply:
Agreed.  Thanks.